packages/workspace/src/browser/workspace-variable-contribution.ts

```diff
@@ -20,6 +20,8 @@ import { ApplicationShell, NavigatableWidget } from '@theia/core/lib/browser';
 import { VariableContribution, VariableRegistry } from '@theia/variable-resolver/lib/browser';
 import { WorkspaceService } from './workspace-service';
 
+export const VAR_WORKSPACE_FOLDER = 'workspaceFolder';
```

Comment:
I am not sure that we need constants. We should not remove variables anymore, since they are APIs now (they can be used in package.json files without referencing constants).

Reply:
Thank you for your review. 
VariableResolveOptions added, and constant reverted.